### DISCUSSION TITLE
WHILE WATCHING VIDEOS IN NETFLIX AMBIENT MODE IS NOT WORKING / (TURBO/BALANCED/SILENT + All OTHER ) pop-up notification is not showing

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1784] [General]
Describe the problems

1.Ambient Mode is not working while watching videos in Netflix (Didn't tested in Games that use Exclusive Full screen/Full screen mode /Borderless windowed and windowed mode)

2.Pop up notification that shows at the bottom middle of screen when a light or perfomance mode changed via shortcut key is not showing over full screen videos that played via NETFLIX / WINDOWS 'MOVIES & TV' video player also (Didn't tested this problem in Games that use Exclusive Full screen/Full screen mode /Borderless windowed and windowed mode)

https://github.com/seerge/g-helper/assets/154586512/7306ea4e-752b-4b83-a425-bacf03dd8f04

*Tested and confirmed the problem exist in both ROG G15 2021 and TUF F15 2023 :(

LOG FILE
[log.txt](https://github.com/seerge/g-helper/files/13762751/log.txt)

@seerge

**Comment by seerge** [comment]
@RAGE-HERO7 while you are watching netflix there is windows protection from making screenshots probably, so ambient mode that calculates colors from screenshot obviously won't work.

That's how things are

**Reply by RAGE-HERO7** [reply]
Oh that how it works ok alright ☹️

So what about the pop notifications not showing up that notifies us while we change LIGHTING /PERFOMANCE modes (This problem only occurs in a fullscreen app or video playing )
I Clearly shows that issue in the video posted above 👆

**Reply by seerge** [reply]
Most probably also blocked by netflix / windows for the same reasons
